cmd/server/main.go

```diff
@@ -250,7 +250,15 @@ func startServer(c *cli.Context) error {
 		return err
 	}
 
-	prometheus.Init(currentNode.Id, currentNode.Type)
+	var promEnv string
+	if conf.PrometheusEnv != "" {
+		promEnv = conf.PrometheusEnv
+	} else if conf.Development {
+		promEnv = "dev"
+	} else {
+		promEnv = "prod"
+	}
+	prometheus.Init(currentNode.Id, currentNode.Type, promEnv)
```

Comment:
```suggestion
	prometheus.Init(currentNode.Id, currentNode.Type, conf.Environment)
```

Reply:
to keep things in one place, put this in `config.go` `NewConfig(...)`:
```
if conf.Development {
  conf.Environment = "dev"
}
```

And I think its fine to leave the default as "" and not assume the environment name.